lib/Config/AppConfig.php

```diff
@@ -83,9 +87,7 @@ public function getConfig($key=null) {
     public function registerAll() {
         $this->registerNavigation();
         $this->registerHooks();
-        // IJob API is fucked up, so silence the code checker
-        $class = '\OCP\BackgroundJob';
-        $class::addRegularTask($this->config['cron']['job'], 'run');
+        $this->jobList->add('OC\BackgroundJob\Legacy\RegularJob', [$this->config['cron']['job'], 'run']);
```

Comment:
Right, the app only works on 10+ too right now and I have no problems to bump it to 11 :)